addons/danger/functions/fnc_brainEngage.sqf

```diff
@@ -29,13 +29,11 @@ params ["_unit", ["_type", -1], ["_target", objNull]];
 // timeout
 private _timeout = time + 0.5;
 
-// ACE3
-_unit setVariable ["ace_medical_ai_lastFired", CBA_missionTime];
-
 // check
 if (
     isNull _target
     || {_unit knowsAbout _target isEqualTo 0}
+    || {(speed _target) > 20}
```

Comment:
why 20? iirc running is like a speed of 11

Reply:
Arbitrary number to be honest.  I observed AI flying slowly in a few different helicopters.  Wanted to pick something where they would suppress when moving slowly, but not when moving too quickly.